libs/pages/services/src/lib/ui/container/container.tsx

```diff
@@ -66,34 +67,35 @@ export function Container(props: PropsWithChildren<ContainerProps>) {
   )
 
   const headerActions = (
-    <>
+    <div className="flex flex-col gap-3">
+      <div className="flex flex-row gap-2">
+        {environment && (
+          <Skeleton width={80} height={32} show={!environment?.mode}>
+            <EnvironmentMode size="xs" mode={environment.mode} />
```

Comment:
For this component, could you add the first letter capitalized only like `Production` & `Development` and others, we had a mix between a cropped name and full name (`PREVIEW` for example) it's weird to keep it

Reply:
https://github.com/Qovery/console/pull/850/commits/d1690f90b3bd575d616cfe5671ce54442e5749e9